src/style.css

```diff
@@ -0,0 +1,67 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-form {
+  width: 100%;
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+input::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  color: #3d4e61;
+}
+
+input {
+  border: none;
+  outline: none;
+  width: 100%;
+}
+
+.input-big {
```

Comment:
add box-sizing for your inputs not on hover or focus, just on default state, I believe it will fix some of your tests